tools/packaging/kata-deploy/Dockerfile

```diff
@@ -4,19 +4,10 @@
 
 # Specify alternative base image, e.g. clefos for s390x
 ARG IMAGE
-FROM ${IMAGE:-registry.centos.org/centos}:7
-ARG KATA_ARTIFACTS=./kata-static.tar.xz
+FROM ${IMAGE:-docker.io/centos}:7
+ARG KATA_ARTIFACTS=./local-build//kata-static.tar.xz
 ARG DESTINATION=/opt/kata-artifacts
 
-COPY ${KATA_ARTIFACTS} ${WORKDIR}
-
-RUN \
-yum -y update && \
```

Comment:
Why did you remove this?

We should always be looking for package fixes, which would come from a yum update.

Reply:
After replacing COPY with ADD, this become necessary. Remove it to save image size. 
See reference https://github.com/kata-containers/kata-containers/pull/5534#discussion_r1009103543

Reply:
If we need fixes, like emergency security fixes, we should update the base image instead of using unpredictable yum update.  In a container world, immutable is important feature. Just my two cents though.

Reply:
And we're working on a different way to address this.

There's been a whole bunch of work in the CCv0 branch, which will end up merged also in the `main` branch, to cache the images used and provide those as part of the release process, so users can easily reproduce the build.

For now, please, leave it as it's.

Reply:
@liyimeng, any further updates here?

Reply:
@fidencio Not sure if I am getting you. Do you want to get back line 11-14, which I removed in this commit?

Reply:
Yes.

Reply:
hmm, I still think replace these with line 20 21 as below is more elegant way to do thing. If you insist, I could put pack line line 13 and 14. 

Honestly, If we do care security, this image should run rootless and something other more proper solution. This update dose not add much value here.